docs/01-start-here/06-faqs.md

```diff
@@ -0,0 +1,23 @@
+#FAQs
+
+## I can see mark-up in my response, but it's not in the application anywhere. Where is it coming from?
+
+Your markup is coming from CAPI and used as-is.
```

Comment:
Yeah, this question was asked originally because we couldn't find a css class in the app html (including cleaners) but makes sense to explicitly mention them